### ISSUE TITLE
new set of pathogenic variants

### ISSUE THREAD

**Issue by brentp** [quinlan-lab/pathoscore#61]
https://content.iospress.com/articles/translational-science-of-rare-diseases/trd027

**Comment by brentp** [comment]
downloadable from here: http://dpv.cmg.med.keio.ac.jp/dpv-pub/variants at bottom of page.

**Comment by jimhavrilla** [comment]
Cool, a Japanese specific set.  Nice find @brentp

**Comment by jimhavrilla** [comment]
Like PrimateAI, no direct link.  I asked Dr. Kenjiro Kosaki, the corresponding author for permission to host it, we'll see what he says.

**Comment by brentp** [comment]
I added a link to the download page

**Comment by jimhavrilla** [comment]
Right, but it is a Javascript link that loads the file.  There is no way to curl or wget.

**Comment by jimhavrilla** [comment]
Dr. Kosaki never wrote me back...maybe a language barrier?  I could try writing him again...
